app/documents/order_fax.rb

```diff
@@ -74,33 +75,41 @@ def recipient_paragraph
 
   def articles_paragraph
     total = 0
-    data = [I18n.t('documents.order_fax.rows')]
+    data = [I18n.t('documents.order_fax.rows').clone]
+    any_order_number_present = false
     each_order_article do |oa|
       price = oa.article_version.price
       subtotal = oa.units_to_order * price
       total += subtotal
-      quantity = if oa.article_version.supplier_order_unit_is_si_convertible
-                   number_with_precision(oa.units_to_order, precision: 3)
-                 else
-                   oa.units_to_order.floor
-                 end
-      data << [oa.article_version.order_number,
-               quantity,
+      order_number = oa.article_version.order_number
+      any_order_number_present = true if order_number
+      data << [order_number,
+               format_units_to_order(oa),
                format_supplier_order_unit_with_ratios(oa.price),
                oa.article_version.name,
                number_to_currency(price),
                number_to_currency(subtotal)]
     end
-    data << [I18n.t('documents.order_fax.total'), nil, nil, nil, nil, number_to_currency(total)]
+
+    total_row = [I18n.t('documents.order_fax.total'), nil, nil, nil, nil, number_to_currency(total)]
+
+    unless any_order_number_present
+      data.map { |row| row.delete_at(0) }
+      total_row.delete_at(1)
+    end
```

Comment:
Removing the `order_number` column when no article has an order_number this way is quite quirky.

Also it's out of scope of the hackathon fork: The same could be done upstream at a later point. This just increases complexity for the final PR and all required rebases on master leading up to it. (The same goes for the corresponding logic in `order_txt`)